File: BOJ/Brute_Force/BJ7568.js
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const [n, ...arr] = input;
const N = Number(n);
const info=[];
const rate = new Array(N).fill(0);

arr.forEach(v => {
  const [weight, height] = v.split(' ').map(Number);
  info.push({weight, height});
});

for(let i in info) {
  for(let j in info) {
    if(i===j) rate[i]++;
    else if(info[i].weight < info[j].weight && info[i].height < info[j].height) {
      rate[i]++;
    }
  }
}
const ans = rate.join(' ');
console.log(ans);

File: BOJ/Brute_Force/BJ7568.cpp
#include <iostream>
using namespace std;

int N;
int weight[55];
int height[55];
int rate[55];

int main() {
  ios::sync_with_stdio(0);
  cin.tie(0);

  cin >> N;

  for(int i=0; i<N; i++) {
    cin >> weight[i] >> height[i];
  }

  for(int i=0; i<N; i++) {
    for(int j=0; j<N; j++) {
      if(i==j) rate[i]++;
      else if(weight[i] < weight[j] && height[i] < height[j]) {
        rate[i]++;
      }
    }
  }
  for(int i=0; i<N; i++) {
    cout << rate[i] << ' ';
  }
}